### DISCUSSION TITLE
2020.2 Java 社招面试整理

### DISCUSSION THREAD

**Discussion by giscus** [wujun234/giscus#8] [Announcements]
# 2020.2 Java 社招面试整理

https://wujun234.com/post/2020-2-Java-she-zhao-mian-shi-zheng-li/

**Comment by oneStepReach** [comment]
看来大厂的生活很繁忙啊？好几年没有出工作总结啦

**Reply by wujun234** [reply]
前两年是太忙了， 这两年是躺平了，肝 Palworld 中 😂
